### ISSUE TITLE
Bulbdog attributes

### ISSUE THREAD

**Issue by Zhinjio** [cadon/ARKStatsExtractor#562]
And obviously, bulbdogs have different stats:

Health
Charge Capacity
Charge Regen
Food
Weight
Charge Emission Range
Movement Speed
Torpidity

Extra fails with red on Stamina, Food and Damage.

**Comment by Zhinjio** [comment]
Also applies to Shinehorn, and probably featherlight, though I've not seen one of those yet.

**Comment by audiopathik** [comment]
Yes, does apply to Featherlight and Glowtail as well and I was looking for the answer to the same issue here, how to add Charge Light emitting creatures to your ASB library and use it to find the ideal breeding partners.

**Comment by cadon** [comment]
Are these new stats just renamed old stats or is their unit different? I.e. is charge capacity and charge regen an absolute number and charge emission range a percentage? Or just all absolute numbers?

**Comment by audiopathik** [comment]
It looks like they are just renamed Stamina, Oxygen and Melee Attack stats since they have the very same symbol next to them and that's the stats they replace for these species.

Charge Capacity and Charge Regen are absolute values, Charge Emission Range is a percentage - just like the stats they replace.

Here are screenshots of Bulbdog, Shinehorn and Featherlight stats:
[https://imgur.com/a/0b8nz](https://imgur.com/a/0b8nz)

They are probably the very same values, just renamed.


I did try to add them to the ASB library the way the OCR feature reads the values (Charge Capacity for Stamina, etc..), since I don't yet have the exact server settings for both Aberration servers I'm playing on most extractions fail though.
